### ISSUE TITLE
Перевірити актуальність панелей в Drupal 8.3

### ISSUE THREAD

**Issue by podarok** [drupal-ukraine/panels#5]
глянути всі issue, реалізувати до панелі в межах контексту, доступ до кожного блоку, блоки які надаються page manager display plugin, panels display plugin, panels cloning (clone, share patch with Dima @podarok )

Блокери незрозуміло, перечитати реліз Drupal 8.3, layout_plugin in core. @dstorozhuk Зібрати META issues по всім модулям

**Comment by podarok** [comment]
TODO: Собрать meta issue по panels, page manager, layout plugin, ctools

**Comment by andypost** [comment]
Начните https://www.drupal.org/node/2852655

**Comment by dstorozhuk** [comment]
Спасибо, @andypost

**Comment by pazhyn** [comment]
There are 4 Active Critical issues at the moment - https://www.drupal.org/project/issues/panels?text=&status=Open&priorities=400&categories=All&version=8.x&component=All
Should we start from the highest priority or the fastest issues to be resolved?

**Comment by dstorozhuk** [comment]
@pazhyn, проверить актуальность - имеется ввиду на сколько нужны будут панели в drupal 8.3 вообще. Ведь планируется вставить в drupal 8.3 layout plugin. И возможно тогда тот бенефит котрый дают панели в связке LP будет в core. И тогда все что делалось через page manager, panels, LP будет делаться только page manager.

**Comment by jorgik1** [comment]
Panels, это UI, + context, Layout предоставляет только API, это уже тестил на 8.3. + к панелям добавили style plugin (пока только patch в статусе needs review), через который можно темизировать регионы и блоки

**Comment by dstorozhuk** [comment]
@jorgik1 , Спасибо за апдейт, я уже тоже потестил. 
Есть ли желание поучаствовать в кодспринте?

**Comment by jorgik1** [comment]
@dstorozhuk Да, конечно, хочу за ре-роллить патч с style plugin, для ветки 8.4.
Просто я написал модуль уже, Panels extra styles, у меня в профиле, для одного из проектов, основанный на style plugin, и хотел бы обновить его позже.

**Comment by andypost** [comment]
Да, хорошо бы добить стили, там только с дефолтовым я не очень согласен

**Comment by andypost** [comment]
выглядит готовым но нужно тестить, как раз для спринта https://www.drupal.org/node/2769099#comment-12049274

**Comment by andypost** [comment]
- https://www.drupal.org/node/2811175
- https://www.drupal.org/node/2824890 *needs work & test modules/system/tests/src/Unit/Transliteration/MachineNameControllerTest.php:18 https://www.drupal.org/docs/8/phpunit
- https://www.drupal.org/node/2660124 layout icons (needs reroll)
